### ISSUE TITLE
please update font_awesome_flutter

### ISSUE THREAD

**Issue by barbalex** [4Q-s-r-o/thumbnailer#7]
When running `flutter pub get` I get:
```
Running "flutter pub get" in capturing...                       
Because thumbnailer 2.0.1 depends on font_awesome_flutter ^9.0.0 and capturing depends on font_awesome_flutter 10.1.0, thumbnailer 2.0.1 is forbidden.
So, because capturing depends on thumbnailer 2.0.1, version solving failed.
```

**Comment by ethael** [comment]
Hello. Sorry for late response. We will update dependencies to up-to-date versions soon after we will upgrade internally to Flutter 3. Thumbnailer depends on native_pdf_renderer, which has a bug that prevents successful build on Flutter3 (https://github.com/ScerIO/packages.flutter/issues/298). We will release new version when it will work with all deps under Flutter 3

**Comment by ethael** [comment]
fixed in ```2.0.2```
